recipes/cotila/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES CXX)
+
+include("${PROJECT_BINARY_DIR}/conanbuildinfo.cmake")
```

This does not really matter here, but this should be `CMAKE_BINARY_DIR`.
A `conanbuildinfo.cmake` file is usually generated once for a build. Not once for each (sub)project
```suggestion
include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```